model/gen-ai/spans.yaml

```
@@ -122,3 +122,65 @@ groups:
       - ref: server.port
         requirement_level:
           conditionally_required: If not default (443).
+
+  - id: span.gen_ai.client.create_agent
+    type: span
+    stability: experimental
+    brief: >
+      Describes GenAI agent creation and is usually applicable when working
+      with agent services.
```

maybe 

```suggestion
      with remote agent services.
```
?

and we'll figure out local agents as we work on frameworks

---

Done